.github/workflows/beam_LoadTests_Python_GBK_Dataflow_Streaming.yml

```diff
@@ -0,0 +1,102 @@
+  cancel-in-progress: true
+
+env:
+  GRADLE_ENTERPRISE_ACCESS_KEY: ${{ secrets.GE_ACCESS_TOKEN }}
+  GRADLE_ENTERPRISE_CACHE_USERNAME: ${{ secrets.GE_CACHE_USERNAME }}
+  GRADLE_ENTERPRISE_CACHE_PASSWORD: ${{ secrets.GE_CACHE_PASSWORD }}
+
+jobs:
+  beam_LoadTests_Python_GBK_Dataflow_Streaming:
+    if: |
+      github.event_name == 'workflow_dispatch' ||
+      github.event_name == 'schedule' ||
+      github.event.comment.body == 'Run Load Tests Python GBK Dataflow Streaming'
+    runs-on: [self-hosted, ubuntu-20.04, main]
+    timeout-minutes: 720
+    name: ${{ matrix.job_name }} (${{ matrix.job_phrase }})
+    strategy:
+      matrix:
+        job_name: ["beam_LoadTests_Python_GBK_Dataflow_Streaming"]
+        job_phrase: ["Run Load Tests Python GBK Dataflow Streaming"]
+    steps:
+      - uses: actions/checkout@v3
+      - name: Setup repository
+        uses: ./.github/actions/setup-action
+        with:
+          comment_phrase: ${{ matrix.job_phrase }}
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          github_job: ${{ matrix.job_name }} (${{ matrix.job_phrase }})
+      - name: Setup environment
+        uses: ./.github/actions/setup-environment-action
+        with:
+          python-version: 3.8
+      - name: Prepare test arguments
+        uses: ./.github/actions/test-arguments-action
+        with:
+          test-type: load
+          test-language: python
+          argument-file-paths: |
+            ${{ github.workspace }}/.github/workflows/load-tests-job-configs/python_GBK_Dataflow_Streaming_2GB_of_100kB_records.txt
+      - name: get current time
+        run: echo "NOW_UTC=$(date '+%m%d%H%M%S' --utc)" >> $GITHUB_ENV
+        # The env variable is created and populated in the test-arguments-action as "<github.job>_test_arguments_<argument_file_paths_index>"
+      - name: run 2GB of 100kB records test
+        uses: ./.github/actions/gradle-command-self-hosted-action
+        with:
+          gradle-command: :sdks:python:apache_beam:testing:load_tests:run
+          arguments: |
+            -PloadTest.mainClass=apache_beam.testing.load_tests.group_by_key_test \
+            -Prunner=DataflowRunner \
+            -PpythonVersion=3.8 \
+            '-PloadTest.args=${{ env.beam_LoadTests_Python_GBK_Dataflow_Streaming_test_arguments_1 }} --job_name=load-tests-python-dataflow-streaming-gbk-3-${{env.NOW_UTC}}' \
+            
+          # // TODO(https://github.com/apache/beam/issues/20403): Skipping some cases because they are too slow.
+          # def TESTS_TO_SKIP = [
+          #   'load-tests-python-dataflow-streaming-gbk-1',
+          #   'load-tests-python-dataflow-streaming-gbk-2',
+          #   'load-tests-python-dataflow-streaming-gbk-4',
+          #   'load-tests-python-dataflow-streaming-gbk-5',
+          # ]
```

Comment:
```suggestion
          # // TODO(https://github.com/apache/beam/issues/20403). Skipping some cases because they are too slow:
          # load-tests-python-dataflow-streaming-gbk-1
          # load-tests-python-dataflow-streaming-gbk-2
          # load-tests-python-dataflow-streaming-gbk-4
          # load-tests-python-dataflow-streaming-gbk-5
```

Reply:
fixed, thanks